api/client/service/update.go

```diff
@@ -30,8 +31,8 @@ func newUpdateCommand(dockerCli *client.DockerCli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.String("image", "", "Service image tag")
-	flags.StringSlice("command", []string{}, "Service command")
-	flags.StringSlice("arg", []string{}, "Service command args")
+	flags.String("command", "", "Service command")
+	flags.String("args", "", "Service command args")
```

Comment:
I'm not thrilled about `args` being a flat string that we do our own lexing on, but I don't really have a better suggestion. At least this is a big improvement from the current approach. The downside is that users will have to double-escape anything that needs escaping.